### ISSUE TITLE
Inconsistent terminology for passcode: "One Time Code" and "Temporary Authentication Code"

### ISSUE THREAD

**Issue by brittag** [cloudfoundry/uaa#750]
In the UAA code that my CF deployment uses, I noticed that the command line says ["**One Time Code**"](https://github.com/cloudfoundry/uaa/search?utf8=%E2%9C%93&q=%22one+time+code%22&type=) while the website says ["**Temporary Authentication Code**"](https://github.com/cloudfoundry/uaa/search?utf8=%E2%9C%93&q=%22temporary+authentication+code%22&type=). This means we're calling the same thing two different phrases, which could be confusing for new users, so being consistent could improve usability.

There's a third inconsistency in that the URL introduces a third term, `passcode`, but that seems a bit more complex to update, and it seems less important than making the UI terms match. If possible though, making that match too (something like `authcode`) could be helpful.

### What did you do?

The command line:

```
cf login -a [ENDPOINT] --sso
API endpoint: [ENDPOINT]

One Time Code ( Get one at https://[SITE]/passcode )> 
```

The web page at `https://[SITE]/passcode`:

![screen shot 2017-12-18 at 4 40 23 pm](https://user-images.githubusercontent.com/391313/34134872-2a5d279e-e412-11e7-9a15-fdbecb12ab42.png)

### What did you expect to see? What goal are you trying to achieve with the UAA?

I suggest making the command line say "Temporary Authentication Code" (instead of "One Time Code") to match the site. "Temporary Authentication Code" is a more descriptive and meaningful name.

I checked out the code to see if this was a simple string update that I could put a PR in for myself (as a technical writer, not a developer), but it looks a bit more complex since there are also tests and docs - happy to give it a shot though if you like.

Thank you!

**Comment by tack-sap** [comment]
Fixed via b7bc35b

Should now be "Temporary Authentication Code"

**Comment by brittag** [comment]
Thank you!
